Webdriver test errors

# Issue Description
We started seeing various errors on travis CI related to webdriver. The error messages are some what consistent around errors with before/after hooks.

<details>
 <summary>Sample Error message</summary>
<pre><code>/home/travis/build/cerner/terra-core/node_modules/webdriverio/build/lib/utils/ReporterStats.js:381
            if (!suiteStats.tests[uid]) throw Error(`Unrecognised test [${runner.title}] for suite [${runner.parent}]`);
                                        ^

Error: Unrecognised test ["before all" hook] for suite [Aligned All]
    at ReporterStats.getTestStats (/home/travis/build/cerner/terra-core/node_modules/webdriverio/build/lib/utils/ReporterStats.js:381:47)
    at ReporterStats.testFail (/home/travis/build/cerner/terra-core/node_modules/webdriverio/build/lib/utils/ReporterStats.js:450:30)
    at BaseReporter.<anonymous> (/home/travis/build/cerner/terra-core/node_modules/webdriverio/build/lib/utils/BaseReporter.js:159:25)
    at emitOne (events.js:116:13)
    at BaseReporter.emit (events.js:211:7)
    at BaseReporter.handleEvent (/home/travis/build/cerner/terra-core/node_modules/webdriverio/build/lib/utils/BaseReporter.js:300:27)
    at Launcher.messageHandler (/home/travis/build/cerner/terra-core/node_modules/webdriverio/build/lib/launcher.js:646:28)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at emit (internal/child_process.js:772:12)
    at _combinedTickCallback (internal/process/next_tick.js:141:11)
    at process._tickDomainCallback (internal/process/next_tick.js:218:9)
npm ERR! Test failed.  See above for more details.</code></pre>
</details>

## Issue Type
<!-- Is this a new feature request, enhancement, bug report, other? -->
- [ ] New Feature
- [ ] Enhancement
- [x] Bug
- [ ] Other

## Expected Behavior
Tests pass on travis CI

## Current Behavior
Tests seem to fail sporadically with errors related to webdriver.io

--- a/wdio.conf.js
+++ b/wdio.conf.js
@@ -19,6 +19,8 @@ const config = {
   baseUrl: `http://${localIP.address()}:${webpackPort}`,
   specs,
 
+  // Travis only has 1 browser instace, set maxInstances to 1 to prevent timeouts
+  maxInstances: process.env.CI ? 1 : wdioConf.config.maxInstances,
   seleniumDocker: {
     enabled: !process.env.TRAVIS,
   },
@@ -36,7 +38,6 @@ const config = {
     global.browser.refresh();
   },
 };
-
 
 config.services = wdioConf.config.services.concat([WebpackDevService]);
 exports.config = config;

